charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -1,3 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-azuredisk-controller-config
+  namespace: {{ .Release.Namespace }}
+{{ include "azuredisk.labels" . | indent 2 }}
+data:
+  config.yaml: |
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
```

Comment:
`apiVersion` should be `disk.csi.azure.com/v1beta2`

Reply:
Ideally CSI drivers should be CO-agnostic, so IMO the configuration file shouldn't be a Kubernetes object (or have its format). Also, using this format would mean we'd need to create a CRD, get the CR created by the user and parse it.